src/main/resources/assets/lambdynlights/lang/de_de.json

```
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
   "lambdynlights.option.block_entities": "Blockentitäten",
-  "lambdynlights.option.entities": "Entitäten",
+  "lambdynlights.option.entities": "Objekte",
   "lambdynlights.option.mode": "Dynamische Lichter",
-  "lambdynlights.option.water_sensitive": "Wasser-sensibel Prüfung",
+  "lambdynlights.option.water_sensitive": "Wasserempfindlichkeit",
   "lambdynlights.toast.first_time": "Automatisch ausgeschaltet, prüfen Sie die Mod- oder Videoeinstellungen nach!",
   "lambdynlights.tooltip.block_entities": "Schaltet Blockentitäten dynamische Beleuchtung ein.",
-  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verfahren der dynamische Beleuchtung des Creepers\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explosierende Creepers.\n- %s: steigende Beleuchtung durch explosierende Creepers.",
-  "lambdynlights.tooltip.entities": "Aktiviert die dynamische Beleuchtung von Entitäten. (Spielerbeleuchtung ist immer aktiviert)",
+  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verhalten der dynamische Beleuchtung des Creepers\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explosierende Creepers.\n- %s: steigende Beleuchtung durch explosierende Creepers.",
+  "lambdynlights.tooltip.entities": "Aktiviert die dynamische Beleuchtung von Objekten. (Spielerbeleuchtung ist immer aktiviert)",
   "lambdynlights.tooltip.mode.1": "Aktiviert dynamische Beleuchtung. Wenn es aktiviert ist wird zB. einem Fackel-haltendem Spieler seine Umgebung beleuchtet.",
   "lambdynlights.tooltip.mode.2": "%s und %s haben eine kleine Verzögerung, aber lösen weniger Lichtupdates aus.",
   "lambdynlights.tooltip.mode.3": "Mit %s sind die Bewegungen von Lichtquellen flüssig.",
-  "lambdynlights.tooltip.tnt_lighting": "Ändert das Verfahren der dynamischer Beleuchtung von TNT.\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung.\n- %s steigende Beuleuchtung.",
-  "lambdynlights.tooltip.water_sensitive": "Aktiviert die Wasser-sensibel Prüfung: einige Objekten werden keinen Licht ins Wasser abgeben."
+  "lambdynlights.tooltip.tnt_lighting": "Ändert das Verhalten der dynamischer Beleuchtung von TNT.\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung.\n- %s steigende Beuleuchtung.",
+  "lambdynlights.tooltip.water_sensitive": "Aktiviert die Wasserempfindlichkeit: einige Gegenstände werden kein Licht im Wasser abgeben."
```

```suggestion
  "lambdynlights.tooltip.water_sensitive": "Aktiviert die Wasserempfindlichkeit: Einige Gegenstände werden kein Licht im Wasser abgeben."
```
A new sentence after a colon should be capitalized: https://www.duden.de/sprachwissen/rechtschreibregeln/doppelpunkt